src/sentry_plugins/github/endpoints/webhook.py

```diff
@@ -71,7 +71,7 @@ def __call__(self, organization, event):
 
             try:
                 with transaction.atomic():
-                    c = Commit.objects.create(
+                    c, created = Commit.objects.get_or_create(
                         repository_id=repo.id,
                         organization_id=organization.id,
                         key=commit['id'],
```

Comment:
@macqueen create_or_update seems to not return the record. I might have been doing something wrong though.